.goreleaser.yaml

```
@@ -0,0 +1,298 @@
+      - goos: darwin
+        goarch: arm
+      - goos: freebsd
+        goarch: arm
+    
+    ldflags:
+      - -s -w
+      - -X cloud-update/src/internal/version.Version={{.Version}}
+      - -X cloud-update/src/internal/version.Commit={{.ShortCommit}}
+      - -X cloud-update/src/internal/version.Date={{.Date}}
+      - -extldflags "-static"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)